src/Core/AdminConsole/OrganizationFeatures/Policies/PolicyValidators/SingleOrgPolicyValidator.cs

```diff
@@ -92,10 +100,20 @@ public async Task<string> ValidateAsync(PolicyUpdate policyUpdate, Policy? curre
     {
         if (policyUpdate is not { Enabled: true })
         {
+            var resultString = new StringBuilder();
+
             var ssoConfig = await _ssoConfigRepository.GetByOrganizationIdAsync(policyUpdate.OrganizationId);
-            return ssoConfig.ValidateDecryptionOptionsNotEnabled([MemberDecryptionType.KeyConnector]);
+            resultString.Append(ssoConfig.ValidateDecryptionOptionsNotEnabled([MemberDecryptionType.KeyConnector]));
```

Comment:
I'm not sure we have a way of displaying multiple errors in the UI, and in any case this would just return one big long string. Better to just return the first validation error that occurs I think. Although you could certainly follow up with design to see if there's some way to display all errors up-front.